worlds/alttp/docs/multiworld_en.md

```diff
@@ -89,16 +89,17 @@ the lua you are using in your file explorer and copy the `socket.dll` to the bas
 
 ##### BizHawk
 
-1. Ensure you have the BSNES core loaded. You may do this by clicking on the Tools menu in BizHawk and following these
+1. Ensure your BizHawk is 2.8 or older, not 2.9 or newer.
```

Comment:
```suggestion
1. Ensure your BizHawk is supported.
    - Windows: BizHawk is 2.8 or older, not 2.9 or newer
    - Linux: Bizhawk is 2.9.1 or newer, not 2.9 or older
```

Reply:
To be clear, 2.9.1+ should work for both Windows and Linux (and 2.9.1+ is the only actually working option on Linux due to LuaInterface not working at all on Linux)

Reply:
> To be clear, 2.9.1+ should work for both Windows [...]

Agreed, but with 2.9.1 not out yet and 2.9.0 currently not working, pointing users to "2.8 or older" will save us a lot of support requests.

> so the luasockets .dll you guys bring in was never actually loaded

I think our new code even works around using the embedded one.
Regardless of that, windows DLL is still required for stuff like snes9x-rr, and some "special sauce" will be required regardless because we support both lua 5.4 and lua 5.1 now. Plus the linux-lua5.4-socket.so is actually being used with 2.9.1+.

Thanks for solving the Linux issue. Will save me a lot of headaches in the future :smiley: